hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieWriteClient.java

```diff
@@ -1163,13 +1155,17 @@ protected void completeLogCompaction(HoodieCommitMetadata metadata, HoodieTable
     tableServiceClient.completeLogCompaction(metadata, table, logCompactionCommitTime);
   }
 
+  public void completeLogCompaction(String compactionInstantTime, HoodieWriteMetadata<O> compactionWriteMetadata, Option<HoodieTable> tableOpt) {
```

Comment:
This is only used by tests.  Could we make this default visibility and visible to tests only?  Also why is this method needed for production code?

Reply:
after I addressed couple of comments from Danny, we don't need these anymore. these were only used for mdt writes.